<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBMAT60TablePanel</class>
 <widget class="QWidget" name="DBMAT60TablePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBMAT60TablePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Initialize AT60 Table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectAll">
       <property name="text">
        <string>SelectAll</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnSelectNone">
       <property name="text">
        <string>SelectNone</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="_cbGimbal">
       <property name="text">
        <string>Gimbal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="_cbCCD">
       <property name="text">
        <string>CCD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="_cbFocus">
       <property name="text">
        <string>Focus</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="_cbFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="_cbSlaveDome">
       <property name="text">
        <string>SlaveDome</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="_cbPlan">
       <property name="text">
        <string>Plan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="_cbInstruction">
       <property name="text">
        <string>Instruction</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
